src/shared/components/msktable/MSKTable.tsx

```diff
@@ -4,13 +4,17 @@ import {observable, computed, action} from "mobx";
 import {observer} from "mobx-react";
 import './styles.scss';
 import {PaginationControls} from "../paginationControls/PaginationControls";
+import {ColumnVisibilityControls} from "../columnVisibilityControls/ColumnVisibilityControls";
+import {CopyDownloadControls} from "../copyDownloadControls/CopyDownloadControls";
 import DefaultTooltip from "../DefaultTooltip";
 
 export type Column<T> = {
     name: string;
     filter?:(data:T, filterString:string)=>boolean;
     sort?:(a:T, b:T, ascending:boolean)=>number;
+    visibility?:ColumnVisibility|ColumnVisibilityFunction;
```

Comment:
what's the use case for ColumnVisibilityFunction? I don't see it used like that here